### DISCUSSION TITLE
Exception when changing color to a shape

### DISCUSSION THREAD

**Discussion by YRobles** [ShapeCrawler/ShapeCrawler#392] [Q&A] [answered]
Hi,
First of all, thanks a lot for this proyect. It is helping me so much.
I'm trying to change the color to a shape:
![image](https://user-images.githubusercontent.com/4245483/203325639-8eae7ae2-65d6-4a7b-9f4c-d55f6e168daa.png)
but I'm getting exception "The parent of this element is null". I'm verified that the shape is found and I just wanna change it the color. Please, could you help me?

**Comment by YRobles** [comment, marked as answer]
Sorry. It seems like something wrong happened with the shape. I tried with another shape and it works fine.

**Reply by AdamShakhabov** [reply]
Hello. Thank you for the feedback🙂

You say the issue is resolved. However, if it is possible, could you share the presentation that has the shape which failed on `.SetHexSolidColor()`? I would like to check this case.

**Reply by YRobles** [reply]
Sorry, I drop the original presentation.. but thanks for your help!
